horace_core/sqw/@data_sqw_dnd/data_sqw_dnd.m

```diff
@@ -42,6 +40,9 @@
             -Inf,-Inf,-Inf,-Inf] % [Inf,Inf,Inf,Inf;-Inf,-Inf,-Inf,-Inf] -- convention if no pixels
         pix = PixelData()      % Object containing data for each pixel
         axis_caption=an_axis_caption(); %  Reference to class, which define axis captions
+        %
+        % returns number of pixels, stored within the PixelData class
+        num_pixels
```

Comment:
Do we need to expose the number of pixels in the parent rather than as `data.pix.num_pixels`?

Reply:
I believe, this should be topmost parameter on sqw, as soon as we got rid of `disp` wrapper. as such, it is equally useful on data_sqw_dnd, as you want to know it during debugging